pkgs/by-name/fa/faugus-launcher/package.nix

```
@@ -0,0 +1,131 @@
+{
+  bash,
+  fetchFromGitHub,
+  gamemode,
+  gamescope,
+  gobject-introspection,
+  gtk3,
+  icoextract,
+  imagemagick,
+  lib,
+  libayatana-appindicator,
+  libcanberra-gtk3,
+  mangohud,
+  nix-update-script,
+  python3Packages,
+  sc-controller,
+  umu-launcher,
+  wrapGAppsHook3,
+  xdg-user-dirs,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "faugus-launcher";
+  version = "1.5.1";
+
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "Faugus";
+    repo = "faugus-launcher";
+    tag = version;
+    hash = "sha256-gCU45Dw9v0RTejQCEHoaC9Vuu1lEU+Wjf+CrWDRvZ6U=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    gtk3
+    libayatana-appindicator
+  ];
+
+  propagatedBuildInputs =
+    with python3Packages;
+    [
+      filelock
+      pillow
+      psutil
+      pygobject3
+      pynput
+      requests
+      vdf
+    ]
```

```suggestion
    (with python3Packages;
    [
      filelock
      pillow
      psutil
      pygobject3
      pynput
      requests
      vdf
    ])
```
Otherwise `with` applies to the second list (although in this case it has no effect).

---

Interesting. I bet there are others with this same issue. Not sure if this warrants a tree wide fix.